requirements.md

```diff
@@ -30,13 +34,15 @@ See also: [SPC Scope](scope.md) for definitions and more information.
 ### Instrument Information
 
 * Enrollment must include display information for the associated instrument.
+
+Q: Should this be a 'must' if we're allowing for per-RP credentials (i.e. as opposed to requiring per-payment-instrument)? I could see it as an optional include in that case.
```

Comment:
Interesting comment. I can imagine multiple use cases:
 * One authentication credential for one instrument.
 * One authentication credential for multiple instruments, where each instrument is independently addressable.
 * One authentication credential per user profile, where individual instruments are not independently addressable. (In which case I agree with your point.)

Those are independent of the ability to have multiple authentication credentials (e.g., phone, desktop) for the same instrument.

Do the above three make sense? Are there others? Is there a single model that enables the above three?

Reply:
See concrete proposal for 1-1 cardinality authentication-to-instrument:
 https://lists.w3.org/Archives/Public/public-payments-wg/2021May/0016.html

This proposal enables "simultaneous enrollment of N instruments for one authentication credential", resulting in N "per-instrument" credentials. It does not support "per-RP credentials." It's not clear to me what that would mean in practice. 

It is not clear that "RP" is the right level of granularity. For example, a person might have multiple identities with the same RP. I also think that "profile-at-PR" is not the right granularity since one might have multiple accounts for a given identity.

Reply:
@stephenmcgruer, I think we can close this thread:
* I do not think we want "per-RP" credentials (e.g., a user might have multiple accounts with an RP).
* There is now a pull request to support dynamic binding to an instrument at authentication time.
* There is a requirement that each instrument/auth credential pair (SPC Credential) be individually addressable.

If you agree, I think you can close the pull request. :)

Thanks!

Reply:
I agree, but for completeness will note:

> I do not think we want "per-RP" credentials (e.g., a user might have multiple accounts with an RP).

At the risk of being a broken-record, 'per-RP' was my lazy way of saying 'cardinality should be up to the RP', which I communicated poorly. (Though I believe webauthn folks also often say 'per-RP' when they really mean 'per account or other state concept that the RP decides on' :D)


I'm rebasing atm, will close the PR if post-rebase everything seems good.